### PR TITLE
graphviz: support style attributes

### PR DESCRIPTION
**Pull request by nrc**
r? @pnkfelix

### CODE REVIEW

File: src/libgraphviz/lib.rs [hunk 1]
```diff
@@ -654,16 +728,26 @@ mod tests {
...
+            }
+        }
     }
 
     impl LabelledGraph {
         fn new(name: &'static str,
                node_labels: Trivial,
-               edges: Vec<Edge>) -> LabelledGraph {
+               edges: Vec<Edge>,
+               node_styles: Vec<Style>) -> LabelledGraph {
```
Comment (pnkfelix) [inline src/libgraphviz/lib.rs]: could you make the `node_styles` an `Option<Vec<Style>>` instead?  In your tests, it seemed annoying to construct the vecs of no-style so frequently.

**Comment by pnkfelix** [comment]
@nrc looks fine.  I trust you to make your own call regarding my suggestion about the API change.  r=me after that is or is not addressed.  :)

**Comment by nrc** [comment]
So I ended up making the ctor take an Option rather than changing the field. I think that gives us the best of both worlds.

**Comment by pnkfelix** [comment]
> So I ended up making the ctor take an Option rather than changing the field. I think that gives us the best of both worlds.

oh great, changing the ctor was actually what I was intending.  :)

**Pull request merged**